core/pom.xml

```diff
@@ -75,8 +75,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
         <configuration>
-          <failOnError>true</failOnError>
-          <failOnWarnings>true</failOnWarnings>
+          <failOnError>false</failOnError>
+          <failOnWarnings>false</failOnWarnings>
```

Comment:
Why is this happening?

Reply:
Whoops, changed that to test since JavaDocs will throw an error when an unchecked throw is documented which is what the Checkstyle check is supposed to throw errors on. Once I get it so the JavaDocs doesn't error on that I'll revert it in a future commit.